packages/plugins/experimental/plugin-script/src/templates/queue.ts

```diff
@@ -0,0 +1,149 @@
+//
+// Copyright 2025 DXOS.org
+//
+/* eslint-disable no-console */
+// TODO(ZaymonFC): Make dxos log available.
+
+// @ts-ignore
+import { createStatic, defineFunction, S, ObjectId, EchoObject, Filter } from 'dxos:functions';
+// @ts-ignore
+import { FetchHttpClient } from 'https://esm.sh/@effect/platform@0.77.2?deps=effect@3.13.3';
+// @ts-ignore
+import { Effect } from 'https://esm.sh/effect@3.13.3';
+
+export default defineFunction({
+  inputSchema: S.Any,
+  outputSchema: S.Struct({ success: S.Boolean }),
+  handler: ({ event, context: { space } }: any) =>
+    Effect.gen(function* () {
+      console.log('Starting queue function', JSON.stringify(event.data));
+
+      console.log('Upserting contact stored schema');
+      const [contactStoredSchema] = yield* Effect.tryPromise({
+        try: () => space.db.schemaRegistry.register([ContactType]),
+        catch: (e: any) => e,
```

Comment:
we should try excluding this again because afaik it should be the default and it's not clean to look at. iirc @yarolegovich added this originally because we were having trouble with errors in functions, but I don't think it should be necessary unless we actually want to do something with the error